pyanaconda/modules/users/users_interface.py

```diff
@@ -100,11 +100,6 @@ def IsRootPasswordSet(self) -> Bool:
         """
         return self.implementation.root_password_is_set
 
-    @emits_properties_changed
-    def SetRootAccountLocked(self, root_account_locked: Bool):
```

Comment:
This is used in the webui tests. Which means they will break with this. I will trigger the tests here to verify :)
Would be nice if we can update the test in the same PR, so that we don't have the broken on master after this.

Reply:
Thanks! I plan to check the code again before the final review, because there is some new code in the Web UI now. I will run every test we have before it will be merged to make sure it doesn't break anything :-)

Reply:
webui tests are automatically running now with each force push, so no worries about triggering these ones.